torch/optim/radam.py

```diff
@@ -8,11 +8,11 @@
     Optimizer,
     _default_to_fused_or_foreach,
     _differentiable_doc,
+    _capturable_doc,
     _dispatch_sqrt,
```

Comment:
Also, could `_dispatch_sqrt` be replaced by `** 0.5`? At least functionally, this should be identical, not sure about the speed...

Reply:
I think you're right, but I'll do that update in a separate PR and update all optimizers

Reply:
It would be nice to be sure if `** 0.5` maps exactly to `torch.sqrt` implementation and not some generic `torch.pow`...